### ISSUE TITLE
Cannot run tests on mac

### ISSUE THREAD

**Issue by petebowden** [theforeman/foreman-ansible-modules#127]
Trying to run tests in my env doesn't seem to work:

TASK [Create/Update/Delete domain] *********************************************
fatal: [tests]: FAILED! => {"changed": false, "msg": "Missing required nailgun module (check docs or install with: pip install nailgun)"}

$ pip list | grep nail
nailgun        0.30.2

I'm using virtualenv and I'm wondering if the issue I'm running into has to do with the ansible_python_interpreter being set to vcr_python_wrapper.py. Is something with my environment being lost when the wrapper intercepts the call to the interpreter?

Any ideas?

**Comment by mdellweg** [comment]
To investigate this, you could dump the environment inside the wrapper to a file. Then we would see, if the neccessary variables for the virtualenv are already lost.
I know this mechanism with the virtualenv works fine with Debian, Ubuntu, but we have seen problems on Arch.
@manuelbonk does this look familiar?

**Comment by manuelbonk** [comment]
Yes, this looks familiar. Just to be clear, the `Create/Update/Delete domain` task should be executed on the `fixtures` and not on the `tests` ansible host (where the `ansible_python_interpreter` is set to `../vcr_python_wrapper.py`).

When I ran into the the above mentioned issue (on Arch Linux) a couple of weeks ago I temporarily 'fixed' it by setting `ansible_python_interpreter` to the absolute path to my venv's python executable. I just re-tested it and somehow this error doesn't occur anymore, even though I removed the `ansible_python_interpreter` option from the `fixtures` host.
